CMakeLists.txt

```diff
@@ -566,11 +568,13 @@ else()
   add_c_flag_if_supported(-Wformat-security C_SECURITY_FLAGS)
   add_cxx_flag_if_supported(-Wformat-security CXX_SECURITY_FLAGS)
 
+  if(0)
```

Comment:
This is required for successful build of master branch.
It's not required for is_hdd() implementation itself.
It's mentioned here [#4290](https://github.com/monero-project/monero/pull/4290)